packages/core/e2e/collection.e2e-spec.ts

```diff
@@ -538,9 +587,17 @@ describe('Collection resolver', () => {
             expect(product?.collections.sort(sortById)).toEqual([
                 {
                     id: 'T_10',
+                    name: 'Electronics Breadcrumbs',
+                    parent: {
+                        id: 'T_1',
+                        name: '__root_collection__',
+                    },
+                },
+                {
+                    id: 'T_11',
                     name: 'Pear Breadcrumbs',
                     parent: {
-                        id: 'T_8',
+                        id: 'T_9',
```

Comment:
A new collection has been created in the second channel, so all ID's are +1. This test also orders by ID, so `T_9` becomes `T_10`, which is then placed at the beginning of the array.

Reply:
Yep that's fine